moodle_rb.gemspec

```diff
@@ -11,12 +11,12 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/jobready/moodle-rb'
   spec.license       = 'MIT'
 
-  spec.files         = `git ls-files -z`.split("\x0")
+  spec.files         = Dir["lib/**/*"] + %w(README.md LICENSE Rakefile)
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
-  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
+  spec.test_files    = Dir['spec/**/*']
```

Comment:
This change necessary because the tests are no longer within the file list above.